frontend/src/components/ProjectMemberTable.vue

```diff
@@ -175,6 +181,18 @@ export default {
         const ownerList: ProjectMember[] = [];
         const developerList: ProjectMember[] = [];
         for (const member of props.project.memberList) {
+          if (
+            !props.activeRoleProvider &&
+            member.roleProvider !== props.project.roleProvider
+          ) {
+            continue;
+          } else if (
+            props.activeRoleProvider &&
```

Comment:
I would also put this block first, it's less readable to start with the negate case